cmd/commands.go

```diff
@@ -297,6 +297,7 @@ provisioning stages.`,
 			fs := flag.NewFlagSet("validate", flag.ExitOnError)
 			fs.String("config", "", "Input configuration file")
 			fs.String("adapter", "", "Name of config adapter")
+			fs.String("envfile", "", "Environment file to load")
```

Comment:
Please add `--envfile` documentation to the `Long` doc as well. See the `run` command for an example of how it should look.